resources/public/api/conf/1.0/employment_expenses_ignore.yaml

```diff
@@ -0,0 +1,96 @@
+post:
+  summary: Ignore Employment Expenses
+  description: |
+    This endpoint enables you to ignore HMRC-provided employment expenses for this user. This endpoint can only be used 
+    after the tax year has ended. A National Insurance number and tax year must be provided.
+    
+    Ignoring an expense will remove any amended expenses and add the "dateIgnored" flag to HMRC-provided expenses.
```

Comment:
The last sentence of the description here is slightly different on the spec:
`Ignoring an expense will remove any amended expenses and add the "dateIgnored" flag to HMRC expenses.`